### DISCUSSION TITLE
Does the current version fix the link tracking issue

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#2761] [Development discussions]
Hello,
I know there was an issue with link tracking after version 2.1.2.  Has this been fixed in the current version?

**Comment by Pacerino** [comment]
Not in the current release (2.1.4).

_At least for me._

**Reply by Pacerino** [reply]
Seems to be fixed with 49cceaa6ca862965448041279fc439ecba163ff8 and probably an upcoming release.

**Comment by jshapiro2** [comment]
@willpower232 any updates on this fix?

**Comment by willpower232** [comment]
Which issue are you referring to? The messages on its page in Github would probably confirm whether it had been fixed or not.

**Reply by Pacerino** [reply]
He's referring to this issue https://github.com/postalserver/postal/issues/2415

**Comment by LotisHunters** [comment]
#2415 has been preventing me from updating too.
It is really exciting to see a new release just come out and a bunch of fixes planned for the next release. Hoping this issue can get fixed so that people that use many tracking domains are able to update to the latest version.

**Comment by jshapiro2** [comment]
@willpower232 Any update on this issue??

**Reply by Pacerino** [reply]
Fixed with https://github.com/postalserver/postal/releases/tag/2.2.0 3 weeks ago
